Tests/Symfony/DependencyInjection/DomainExtensionTest.php

```diff
@@ -14,14 +14,31 @@ public function testItAddsDoctrinePostPersistListenerToContainer()
 
         $config = [[
             'persist_listeners' => [
-                'doctrine' => ['default'],
+                'doctrine' => ['default', 'custom_doctrine'],
             ],
         ]];
 
         $container = new ContainerBuilder();
         $extension->load($config, $container);
 
+        $array = [
+            "biig_domain.post_persist_listener.doctrine_default" => [
+                [
+                    "connection" => "default"
+                ]
+            ],
+            "biig_domain.post_persist_listener.doctrine_custom_doctrine" => [
+                [
+                    "connection" => "custom_doctrine"
+                ]
+            ]
+
```

Comment:
please remove this space :) . And all is ok! (you can execute php-cs-fixer here https://github.com/biig-io/DomainComponent/blob/master/.php_cs.dist )